src/NHibernate/Persister/IPersister.cs

```diff
@@ -5,7 +5,7 @@
 namespace NHibernate.Persister
 {
 	// TODO 6.0: Make this public and make IEntityPersister and ICollectionPersister derive it.
```

Comment:
Comment now slightly obsolete:

```suggestion
	// TODO 6.0: Make IEntityPersister and ICollectionPersister derive it.
```